kubernetes-manifests/config.yaml

```diff
@@ -36,7 +36,7 @@ kind: ConfigMap
 metadata:
   name: demo-data-config
 data:
-  USE_DEMO_DATA: "True"
+  USE_DEMO_DATA: "False"
```

Comment:
I assume this should be reverted before merging?

Reply:
Yep, accidentally included it in the commit